config/locales/views/admin/en.users.yml

```diff
@@ -23,3 +25,9 @@ en:
       update:
         error: There was an error updating the user. %{message}
         success: The user has been updated successfully.
+      suspend:
+        success: User suspended
+      unsuspend:
+        success: User re-activated
+      unexpire:
+        success: User un-expired
```

Comment:
I think we need to remove the hyphens here, reactivate and unexpired do not have them.